SilentPatchIII/SilentPatchIII.cpp

```diff
@@ -781,7 +782,7 @@ namespace GenerateNewPickup_ReuseObjectFix
 			mov		edi, eax
 
 			push	edi
-			call	[WorldRemove]
+			call	offset WorldRemove
```

Comment:
This is probably wrong.

Reply:
I mean that the call is supposed to dereference `WorldRemove` and call that, `call offset` will probably attempt to call the address of this variable.